example/migrations/src/main/scala/SampleCodegen.scala

```diff
@@ -17,20 +17,22 @@ object SampleCodegen{
         H2Driver.defaultTables.filter(t =>
           tableNames.contains(t.name.name))))
       val latest = mm.latest
-      List( "v" + latest, "latest" ).foreach{ version =>
-        val pkg = "datamodel." + version + ".schema"
-        val folder = System.getProperty("user.dir")+"/app/src/main/scala"
-        val generator = new SourceCodeGenerator(model) {
-          override def packageCode(
-            profile: String, pkg: String, container: String) : String =
-            super.packageCode(profile, pkg, container) + s"""
+      List("/app", "/migrations").foreach { proj =>

```

Comment:
temporary sounds good.
Here is the project relationships I imagine:
project migrations depends on project generated code
project example app depends on project generated code
project migration manager runs project migrations, which triggers code generation into project generated code (only). it doesn't depend on project migrations or project generated code, which means if either of them are broken you can still run the migration manager to e.g. delete broken generated code